### DISCUSSION TITLE
Yesterday YES but NOT now calls SP-API

### DISCUSSION THREAD

**Discussion by AlbertoCamachoMunoz** [amzn/selling-partner-api-models#3318] [General]
Yesterday I was making API calls using postman, all perfect response, correct results until suddenly ERROR
Without having changed absolutely anything

    "errors": [
        {
            "message": "Access to requested resource is denied.",
            "code": "Unauthorized",
            "details": ""
        }
    ]

I dont have a idea what is the problem, I have been seeing problems related to the **RateLimit**, but I can't find anything related to this problem.
Thanks in advance

**Comment by Grant3W** [comment]
Have you changed your IAM ARN recently? This tripped me up initially. As even when you change it, the old one continues to work for 24 hours (I believe).

**Reply by AlbertoCamachoMunoz** [reply]
First thx for your quick response.
Corrected, the problem was that I have 2 applications A and B, which do exactly the same thing and the same user for both, in postman this user was configured to use application A, but suddenly as I say in the statement, it stops working , doing tests this morning I managed to connect this user to application B through postman, the problem seems to have been solved, but I don't understand what happened from one call to another to produce this change
Thank you very much for answering
